zookeeper/src/main/java/com/linecorp/armeria/server/zookeeper/package-info.java

```diff
@@ -14,14 +14,14 @@
  * under the License.
  */
 /**
- * Automatically server registration by using ZooKeeper.
- * {@link com.linecorp.armeria.server.zookeeper.ZooKeeperUpdatingListener} can automatically register a
- * server to a ZooKeeper cluster. The registered ZooKeeper node is EPHEMERAL, so when server stops or a network
- * partition occurs, the underlying ZooKeeper session will be closed and the node will be automatically removed
- * by ZooKeeper cluster. As a result, the clients that use a ZooKeeperEndpointGroup will be notified and they
- * will update their endpoint list automatically, so that they do not attempt to connect to the unreachable
- * servers.
+ * Provides automatic server registration and service discovery feature with ZooKeeper.
```

Comment:
`Automatic service registration and discovery with <a href="https://zookeeper.apache.org/">ZooKeeper</a>.`